mods/callmanager/src/service/call.ts

```diff
@@ -35,15 +36,15 @@ export default async function (
     throw new FonosError("invalid e164 number");
 
   const response = new CallResponse();
-  response.setDuration(0);
+  response.setRef(nanoid());
 
   // Removing the "+" sign
   const from = request.getFrom().replace("+", "");
   const to = request.getTo().replace("+", "");
 
   const variables = !request.getWebhook()
     ? {DID_INFO: from}
```

Comment:
@wardner 

It looks like you missed adding REF in line 46 if the webhook is undefined.

Reply:
This is solved in the next commit. #128